packages/flutter/lib/src/material/pickers/calendar_date_picker.dart

```diff
@@ -599,14 +741,27 @@ class _MonthPickerState extends State<_MonthPicker> {
               ],
             ),
           ),
-          _DayHeaders(),
```

Comment:
What is the reason for moving this into the PageView?

Reply:
I wanted the headers to scroll with the days, so you could keep their context while they were scrolling. Looking at the spec, they have the headers moving with the dates as well (in addition they also have the nav controls scrolling, which seemed wrong to me).

Reply:
Looks like you are right about the spec, but yeah I agree that the nav controls shouldn't scroll